tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
+| Screen-Space Reflections| ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Screen-Space Ambient    | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| Occlusion (SSAO)        |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Screen-Space            | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| Indirect Lighting (SSIL)|                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Signed Distance Field   | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| Global Illumination     |                          |                          |                          |
+| (SDFGI)                 |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Glow                    | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Adjustments             | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom post-processing  | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
+| with fullscreen quad    |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom post-processing  | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| with CompositorEffects  |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Antialiasing
+~~~~~~~~~~~~
+
+See :ref:`doc_3d_antialiasing` for more information.
+
++-------------------+--------------------------+--------------------------+--------------------------+
+| Feature           | Compatibility            | Mobile                   | Forward+                 |
++===================+==========================+==========================+==========================+
+| MSAA              | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
```

Comment:
```suggestion
| MSAA              | ⚠️ Supported (3D only).            | ✔️ Supported (2D + 3D).            | ✔️ Supported (2D + 3D).            |
```

Reply:
Split this into two rows for 2D and 3D MSAA instead.